addons/medical/functions/fnc_isInjured.sqf

```diff
@@ -0,0 +1,22 @@
+#include "..\script_component.hpp"
+/*
+ * Author: DartRuffian
+ * Checks if a unit needs treatment.
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ *
+ * Return Value:
+ * Does unit need treatment <BOOL>
+ *
+ * Example:
+ * ACE_player call ace_medical_fnc_isInjured
+ *
+ * Public: Yes
+ */
+
+params [["_unit", objNull, [objNull]]];
+
+if !(alive _unit) exitWith {false};
+
+_unit call EFUNC(medical_ai,isInjured)
```

Comment:
`EFUNC(medical_ai,isInjured)` only checks if the unit has broken legs, it doesn't check for broken arms. It also returns `true` if the unit is unconscious.

What purpose should this new function serve?
As it is atm, imo you are asking it to do too much, making it not very useful in my eyes.

Reply:
> What purpose should this new function serve?

For my use-case, it was checking for "non-normal" medical state. I.e. `true` if uncon, has wounds, etc.
It was for a medevac script and I only wanted the action to show if the unit was actually injured.

Reply:
> Should the medical_ai function check for broken arms as well?

No, it was intentionally done like that for (to me unknown) reason, although my guess would be that 1) AI don't have a lot of splints, 2) AI don't suffer from AI sway as players do (so don't waste them) and 3) might have simplified the AI medic state machine by not having to add a priority system for which limbs to splint (is now a moot point, because I revamped the state machine quite a bit).

What is the AI function completely omits is blood level and HR checking. I'll need to check whether this is intentional or not, but regardless the API function needs to check for the blood level at the very least. I'd imagine that having a high/low HR should also make the API return `true`, but then there might be certain scenarios where you simply don't care.

That leads me to: what's the formal definition of "non-normal"?

Reply:
I'd define "non-normal" as a state differing from the state set by `initUnit` / `fullHeal`

Reply:
Are you suggesting that should be added in the check as well?